todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/activities/map/MapsActivity.java

```diff
@@ -35,12 +35,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public void onMapReady(GoogleMap googleMap) {
-        mMap = googleMap;
+        map = googleMap;
 
-        // Add a marker in Sydney and move the camera
-        LatLng sydney = new LatLng(-34, 151);
-        mMap.addMarker(new MarkerOptions().position(sydney).title(toDoContent.getTitle()));
-        mMap.moveCamera(CameraUpdateFactory.newLatLng(sydney));
+        LatLng latLng = new LatLng(-34, 151);
```

Comment:
actually the idea is to pass different locations right? not always the same. thats the point of this.